_posts/2020-07-21-react-native-tracker-0-1-0-released/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Snowplow React Native Tracker 0.1.0 released"
+title-short: Snowplow React Native Tracker 0.1.0
+tags: [snowplow, react-native, tracker]
+author: Colm
+category: Releases
+permalink: /blog/2020/07/21/snowplow-react-native-tracker-0.1.0-released/
+discourse: false
+---
+
+We are pleased to announce the first production-ready release of the [Snowplow React Native Tracker](https://github.com/snowplow-incubator/snowplow-react-native-tracker), version 0.1.0. The React Native tracker imports the native Snowplow iOS and Android trackers to React Native, as Native Modules. Much of the functionality of the native versions of the tracker is available.
+
+Read on below for:
+
+1. [Key features](#features)
+2. [Changes since the alpha release](#changes)
+3. [Quickstart guide](#quickstart)
+4. [Feedback and next steps](#feedback)
+5. [Documentation](#documentation)
+6. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="features">1. Key features</h2>
+
+This first production-ready release of the Snowplow React Native tracker has the following key features, closely in line with the existing functionality of our iOS and Android trackers:
+
+- Automatic tracking: The tracker includes automatic tracking of the mobile context, screen context and session context, as well as installs and lifecycle events. All are configurable on tracker initialisation.
+
+- Standard events: Screen View, Page View, and Structured events are available.
+
+- Custom tracking: Custom Self-Describing events and contexts are available.
+
+- Setting subject data: The subject can be set using the `setSubjectData()` method.
+
+
+<h2 id="changes">2. Changes since the alpha release</h2>
```

Comment:
Should this be included in the blog post at all? 

I put it in the discourse post as the audience for it includes those that have been on the alpha. But as a 'first release' announcement it feels weird to talk about the alpha.

Reply:
Good point, let me rephrase that bit slightly!

Reply:
@colmsnowplow what do you think? I have left it in but moved it towards the end and also rephrased it slightly...

Reply:
Love it.